Assets/Scripts/UI/XboxRemap.cs

```diff
@@ -47,10 +49,36 @@ public void InitiateButton(int passed)
         button = GameObject.Find("Managers").GetComponent<InputManager>().buttons[passed].xboxButton;
         keyName = GameObject.Find("Managers").GetComponent<InputManager>().buttons[passed].xboxButton.ToString();
     }
-    //Curently not working
+    public void Remaping()
+    {
+        StartCoroutine(timerRemaping());
+    }
     public void SetButton(XboxController.XboxButton passed)
     {
+        List<string> temp = new List<string>();
+        temp = GameObject.Find("Player 1 Camera").GetComponentInChildren<PauseMenu>().xboxCodes;
```

Comment:
you dont need to initialize the list if you're setting it to the return value of a method which initializes a list
just do
`List<string> temp = GameObject.Find("Player 1 Camera").GetComponentInChildren<PauseMenu>().xboxCodes;`
not sure if it makes sense to store a reference to the pause menu for this but it seems like it should be referenced on the remapping class